src/Microsoft.AspNetCore.Localization/IRequestCultureFeature.cs

```diff
@@ -16,7 +16,7 @@ public interface IRequestCultureFeature
         /// <summary>
         /// The <see cref="IRequestCultureProvider"/> that determined the request's culture information.
         /// If the value is <c>null</c> then no provider was used and the request's culture was set to the value of
-        /// <see cref="RequestLocalizationOptions.DefaultRequestCulture"/>.
+        /// <see cref="Builder.RequestLocalizationOptions.DefaultRequestCulture"/>.
```

Comment:
`RequestLocalizationOptions` is located under `Microsoft.AspNetCore.Builder` which isn't referenced here.